### ISSUE TITLE
Add build documentation

### ISSUE THREAD

**Issue by surajssd** [virtee/sevctl#110]
For me to build `sevctl` on Ubuntu I had to install the following OS packages:

```bash
sudo apt install -y pkg-config libssl-dev asciidoctor
```

And then I could build using the following command:

```bash
cargo build
```

**Comment by surajssd** [comment]
Normally built binary has dependency on the following system libs:

```console
$ ldd target/release/sevctl
        linux-vdso.so.1 (0x00007ffed59d1000)
        libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f5d6d8c1000)
        libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f5d6d47e000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f5d6d45e000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f5d6d236000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f5d6dbea000)
```

To be able to do static build I had to also install the `musl-tools` on top of the existing packages:

```bash
sudo apt install -y pkg-config libssl-dev asciidoctor musl-tools
rustup target add x86_64-unknown-linux-musl
cargo build --release --target x86_64-unknown-linux-musl
```

And using the changes in PR #112, I can now do static build:

```console
$ ldd target/x86_64-unknown-linux-musl/release/sevctl
        statically linked
```

**Comment by tylerfanelli** [comment]
@surajssd Apologies for delay, addressed in #157
